### ISSUE TITLE
Inverted Colors of icons in Nav Bar

### ISSUE THREAD

**Issue by n1tr0-5urf3r** [Icinga/icingaweb2#3181]
## Expected Behavior

Icons from shared Navigation Elements should be displayed as they are.

## Current Behavior

Colors of icons from external sources are displayed with inverted colors (negative)

## Possible Solution

Don't display them in negative i guess, Maybe could frame them if there would be contrast issues?

## Steps to Reproduce (for bugs)

1. Create shared navigation element
2. Set icon (i.e. img/icons/snort.png, the snort.org favicon)
3. Colors in Navigation bar on the left are inverted

## Context

Just a visual issue, my company's logo is displayed in negative. Also the icon of Snort (a pig https://www.snort.org/) cant really be distinguished from the rest because it gets nearly the same color as the background.

## Your Environment

* Icinga Web 2 version and modules (System - About):
Icinga Web 2 Version
    2.5.0
* Version used (`icinga2 --version`):
icinga2 - The Icinga 2 network monitoring daemon (version: r2.8.0-1)
* Operating System and version:
Debian 8
* Enabled features (`icinga2 feature list`):
Enabled features: api checker command ido-mysql mainlog notification perfdata

**Comment by gryphon85** [comment]
The problem only occurs in Firefox (57.0) and Chrome (version 62.0.3202.94), but not in Internet Explorer (version 11.0.9600.18837).

**Comment by Al2Klimov** [comment]
Hello @n1tr0-5urf3r and thank you for reporting!

## TL;DR

Confirmed.
I'm afraid this works as intended.

## ~TL;DR~

E.g. one of our legacy icons `img/icons/edit.png` is clearly a black pencil on a transparent background. If I choose it as nav item icon, it's being displayed light-grey on dark-grey.
This keeps all before-dark-sidebar icons compatible.

Best,
AK

**Comment by nilmerg** [comment]
@Al2Klimov Well, while it works as intended it is clearly wrong how it works. :stuck_out_tongue: 

The inversion is applied to any icon. While that's fine for our builtin icons we're using in the default menu, it's not for any custom icon. We need to differentiate here.

I guess it's fine to invert everything that's clearly our own builtin stuff. Everything else must be kept as is and it's the user's responsibility to adjust the icon if it doesn't fit the new background anymore.

How to differentiate what's our stuff?

- All font icons
- Everything under `public/img/icons` (Possibly except those with `_white` it its name :thinking: )

Custom icons should not be placed under `public/img/icons` anyway. Any other path under `public` should also work. (e.g. `public/our/own/icons/snort.png` referenced as `our/own/icons/snort.png` in the nav-item's configuration @n1tr0-5urf3r )
